MAUI/ImageEditor/events.md

```diff
@@ -64,6 +64,56 @@ N> This is common for Shape and Text annotations.
 
 {% endtabs %}
 
+## ID Support for annotation event
```

Comment:
## ID support for annotation
In ImageEditor, a unique ID is generated for all annotations (Text, Shapes, Paths, and CustomViews) when they are added to the image editor. You can retrieve this unique ID from the ItemsSelected event arguments or from the serialized JSON.

## Select a particular annotation programmatically using annotation ID
By passing the unique ID of an annotation to the SelectAnnotation method of SfImageEditor, you can select the particular annotation programmatically.